packages/console/src/scss/_colors.scss

```diff
@@ -69,6 +81,7 @@
   --color-primary: var(--color-primary-40);
   --color-on-primary: var(--color-all-100);
   --color-error: var(--color-error-40);
+  --color-success: var(--color-success-70);
```

Comment:
These changes in `_colors.scss` have already been included in #550. Please rebase and remove

Reply:
updated